### PR TITLE
Add support for Digilent Digital Discovery and Analog Discovery 2

### PR DESCRIPTION
**Pull request by ultraembedded**
Add support for Digilent Digital Discovery and Analog Discovery 2 (openFPGALoader -c digilent_ad).
I have tested this on my Digilent Digital Discovery, but I'm reasonably sure this works on the Digilent Analog Discovery 2 as well.

### CODE REVIEW

**Comment by trabucayre** [comment]
Thanks for this PR.
Just some remarks:
- your update point an error in high bits configuration: FT4232H has 4 interfaces but with only with low bits, FT232H has only one interface but with both high and low bits. So this test is wrong: `&& _ftdi->type != TYPE_232H` has to be suppressed and consequently your `mpsse_bit_config` update is not required.
- you didn't add a board entry?
rest see bellow

File: README.md [hunk 1]
```diff
@@ -50,6 +52,7 @@ __Supported (tested) FPGA:__
 * Lattice [ECP5 (25F, 5G 85F, CrossLink-NX (LIFCL-40)](https://www.latticesemi.com/en/Products/FPGAandCPLD/CrossLink-NX) (SRAM and Flash)
 * Xilinx Kintex 7 [xc7k325t](https://www.xilinx.com/products/silicon-devices/fpga/kintex-7.html#productTable) (SRAM)
 * Xilinx Artix 7 [xc7a35ti, xc7a50t, xc7a75t, xc7a100t, xc7a200t](https://www.xilinx.com/products/silicon-devices/fpga/artix-7.html) (memory and spi flash)
+* Xilinx Spartan 6 [xc6slx25](https://www.xilinx.com/products/silicon-devices/fpga/spartan-6.html) (memory)
 * Xilinx Spartan 6 [xc6slx45](https://www.xilinx.com/products/silicon-devices/fpga/spartan-6.html) (memory)
```
Comment (trabucayre) [inline README.md:56]: add xc6slx25 before xc6slx45 (`[xc6slx25, xc6slx45]`) in existing entry

**Comment by ultraembedded** [comment]
Ah, ok.
i wasn’t sure if your check for the FT232H part was deliberate or not, and I didn’t want to risk breaking the other boards, hence the additional config flag.

**Comment by trabucayre** [comment]
No, it's a real error. Could add this fix in your PR ?
Thanks

**Comment by ultraembedded** [comment]
Yep. See my 2nd commit above.

**Comment by trabucayre** [comment]
LGTM. 
Just could you update already existing spartan6 line instead of adding a new one ? After that it's Okay
Thanks

**Comment by ultraembedded** [comment]
README modification is now done!

**Comment by trabucayre** [comment]
Applied to master.
Thanks

**Pull request merged**